src/Stripe.net/Entities/Issuing/Cards/Card.cs

```diff
@@ -50,6 +50,13 @@ public class Card : StripeEntity<Card>, IHasId, IHasMetadata, IHasObject
         [JsonProperty("currency")]
         public string Currency { get; set; }
 
+        /// <summary>
+        /// The card’s CVC. For security reasons, this is only available for virtual cards, and will
+        /// be omitted unless you explicitly expand it in the request.
+        /// </summary>
+        [JsonProperty("cvc")]
+        public string Cvc { get; set; }
```

Comment:
in the api ref this is expandable, but I'm not sure what it would expand to? do we need to make this expandable here?

Reply:
No we don't. Expandable here means "includable" which means the field is not present unless you expand it.

Reply:
ack

Reply:
I didn't realize that some scalar fields could be expanded.